crates/iceberg-catalog/src/api/management/mod.rs

```diff
@@ -314,12 +314,10 @@ pub mod v1 {
         pub created_at: chrono::DateTime<chrono::Utc>,
         /// Date when the tabular was deleted
         pub deleted_at: chrono::DateTime<chrono::Utc>,
-        /// Kind of deletion, default or purge
-        pub deleted_kind: DeleteKind,
```

Comment:
I think we should at expiration date as well - its one of the most important things to show for the user, so that he knows how much time is left to undrop

Reply:
added expiration_task_id & expiration_date